vector/src/main/res/layout/adapter_item_vector_message_code.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i suspect it is the same layout than the text message

Reply:
It's different from adapter_item_vector_search_message_text_emote_notice.xml in three ways:  
i) wraps the TextView in a HorizontalScrollView
ii) monospace font
iii) no auto-linking of URLs